lib/middleware/configMiddleware.ts

```diff
@@ -76,6 +76,15 @@ export async function loadConfigMiddleware(
     }
   };
 
+  if (
```

Comment:
Could we add this to the existing `SKIP_CONFIG_VALIDATION` object? I don't remember exactly how that works, but it seems like the same use case as this.

Reply:
`hs account auth` is a special exception, because we want to validate the config if it exists (ie. if we're just authorizing a new account). If it doesn't exist (ie. we're running the command for the first time to create the config), we don't want to validate the config, which is what this conditional stipulates.

Reply:
Ah ok, thanks. Makes sense 👍